test/e2e/main/scaling_test.go

```diff
@@ -9,79 +9,62 @@ import (
 	tutils "github.com/infinispan/infinispan-operator/test/e2e/utils"
 )
 
-// TestPermanentCache creates a permanent cache the stop/start
-// the cluster and checks that the cache is still there
-func TestPermanentCache(t *testing.T) {
+// TestGracefulShutdownWithTwoReplicas creates a permanent cache with file-store and any entry,
+// shutdowns the cluster and checks that the cache and the data are still there
+func TestGracefulShutdownWithTwoReplicas(t *testing.T) {
 	t.Parallel()
 	defer testKube.CleanNamespaceAndLogOnPanic(t, tutils.Namespace)
 
-	cacheName := "test"
-	// Define function for the generic stop/start test procedure
-	var createPermanentCache = func(ispn *ispnv1.Infinispan) {
-		client_ := tutils.HTTPClientForCluster(ispn, testKube)
-		tutils.NewCacheHelper(cacheName, client_).CreateWithDefault("PERMANENT")
-	}
+	// Cache definitions
+	defaultCache := "default-cache"
+	defaultCacheConfig := `<distributed-cache name="` + defaultCache + `" mode="SYNC" statistics="true">
```

Comment:
For ease of reading I think we should simplify this to just `<distributed-cache name=" + defaultCache + "/>"

Reply:
Unfortunately I'm not following up. Could you elaborate?

Reply:
I got the markdown wrong, causing the config to be hidden. Hopefully the original comment is a bit clearer now.

tldr; Lets remove the unnecessary attributes and elements from the cache definition to make the test as simple as possible.

Reply:
Replaced as suggested. Even though it would be nice to have option to define caches in some friendly way in the code. Using xml in any code is just painful. But so is that the cache doesn't have defaults for some features it nags you to have defined (encoding, fetch-state for persistent caches).